apis/validator/duties/proposer.yaml

```diff
@@ -0,0 +1,31 @@
+get:
+  tags:
+    - MinimalValidatorApi
+    - Validator
+  summary: "Get block proposers duties"
+  operationId: "getProposerDuties"
+  description: "Request beacon node to provide all validators that are suppose to propose block in given epoch"
+  parameters:
+    - name: epoch
+      in: path
+      required: true
+      schema:
+        $ref: '../../../beacon-node-oapi.yaml#/components/schemas/Uint64'
+  responses:
+    "200":
+      description: Success response
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              data:
+                type: array
+                items:
+                  $ref: ../../../beacon-node-oapi.yaml#/components/schemas/ProposerDuty
```

Comment:
This array is fixed size. Can we and should we designate that here?

Reply:
We could indicate with same mininimum and maximum props. Should I put mainnet values?

Not really sure how that will impact client code generation (you can generate code to invoke api from spec)? It might set fixed limitations and break if you try to use generated code to invoke api on BN running minimal spec :thinking:

Reply:
no worries